app/views/content_items/topical_event_about_page.html.erb

```diff
@@ -8,9 +8,15 @@
 
 <div class="grid-row">
   <div class="column-two-thirds">
-    <%= render "components/contents-list-with-body", {
-      contents: @content_item.contents
-    } do %>
+    <% if @content_item.show_content_list? %>
```

Comment:
This might be better implemented in the `contents-list-with-body` component as there's already a check for `@content_item.contents.any?`

Reply:
Yeah definitely - I've noted it's not an ideal implementation in the PR description, and would need to investigate other formats using this component before making changes to it in prod.